Indicators/IndicatorExtensions.cs

```diff
@@ -330,6 +330,26 @@ public static SimpleMovingAverage SMA<T>(this IndicatorBase<T> left, int period,
 
         /// The methods overloads bellow are due to python.net not being able to correctly solve generic methods overload
 
+        private static dynamic GetIndicatorAsManagedObject(PyObject indicator)
+        {
+            var indicatorManagedObject = indicator.SafeAsManagedObject();
+
+            if (indicatorManagedObject as PythonIndicator == null)
+            {
+                return indicatorManagedObject;
+            }
+
+            indicator.TryConvert(out PythonIndicator pythonIndicator);
+            pythonIndicator?.SetIndicator(indicator);
+
+            if (pythonIndicator == null)
```

Comment:
- Based on the boolean returned by `TryConvert` the null checks could be avoided I think?
- Wouldn't the TryConvert here cover the case above too? `if (indicatorManagedObject as PythonIndicator == null)`, when could `pythonIndicator == null` after the try convert if we already passed the check above?